README.rst

```diff
@@ -4,14 +4,14 @@ phpspec
 The main website with documentation is at `http://www.phpspec.net <http://www.phpspec.net>`_.
 
 .. image:: https://travis-ci.org/phpspec/phpspec.svg?branch=master
-   :target: http://travis-ci.org/phpspec/phpspec
+   :target: https://travis-ci.org/phpspec/phpspec
    :alt: Master Travis Build Status
 
-.. image:: https://scrutinizer-ci.com/g/phpspec/phpspec/badges/quality-score.png?b=master
+.. image:: https://img.shields.io/scrutinizer/g/phpspec/phpspec.svg
    :target: https://scrutinizer-ci.com/g/phpspec/phpspec/build-status/master
    :alt: Master Scrutinizer Quality Score
 
-.. image:: https://ci.appveyor.com/api/projects/status/wce4nun9re76ocp6/branch/master?svg=true
+.. image:: https://img.shields.io/appveyor/ci/ciaranmcnulty/phpspec.svg
```

Comment:
You can use https://img.shields.io/appveyor/ci/ciaranmcnulty/phpspec/master.svg
`2.5` branch works too, and other names return "invalid" state.